lrpar/src/lib/ctbuilder.rs

```diff
@@ -184,6 +184,16 @@ pub enum RustEdition {
     Rust2021,
 }
 
+impl RustEdition {
+    fn to_variant_tokens(&self) -> TokenStream {
```

Comment:
Can just be `self` because copy.

Reply:
Fixed in [294ce8e](https://github.com/softdevteam/grmtools/pull/505/commits/294ce8e1df172c1c60e45992c8dfb6ffdb426b07)